paimon-core/src/main/java/org/apache/paimon/table/system/StatisticTable.java

```diff
@@ -72,7 +72,7 @@ public class StatisticTable implements ReadonlyTable {
                             new DataField(1, "schema_id", new BigIntType(false)),
                             new DataField(2, "mergedRecordCount", new BigIntType(true)),
                             new DataField(3, "mergedRecordSize", new BigIntType(true)),
-                            new DataField(4, "colstat", SerializationUtils.newStringType(true))));
```

Comment:
please don't modify this, it is better to keep compability.

Reply:
addressed